packages/protocol/contracts/governance/Election.sol

```diff
@@ -763,12 +746,14 @@ contract Election is
       votes.total.eligible.update(group, newVoteTotal, lesser, greater);
     }
 
-    CachedVotes storage cachedVotes = cachedVotesByAccount[account];
-    cachedVotes.totalVotes -= Math.min(cachedVotes.totalVotes, value);
-    cachedVotes.cachedVotesPerGroup[group] -= Math.min(
-      cachedVotes.cachedVotesPerGroup[group],
-      value
-    );
+    if (allowedToVoteOverMaxNumberOfGroups[account]) {
+      CachedVotes storage cachedVotes = cachedVotesByAccount[account];
+      cachedVotes.totalVotes -= Math.min(cachedVotes.totalVotes, value);
+      cachedVotes.cachedVotesPerGroup[group] -= Math.min(
```

Comment:
Can/should we just set this to the current number of votes for this account for this group? Believe that should be correctly updated after the first part of this function, right?